### DISCUSSION TITLE
Recruiting alpha testers for Production Decomposition

### DISCUSSION THREAD

**Discussion by isc-pbarton** [intersystems/git-source-control#519] [Announcements]
We'd like to announce an upcoming feature of git-source-control that is relevant to anybody doing work with IRIS Interoperability productions: Production Decomposition. When Production Decomposition is enabled, multiple developers may edit the same interoperability production in the same namespace without overwriting or blocking each other’s work. This resolves a major obstacle that has prevented organizations from source-controlling their interoperability productions.

We are looking for alpha testers who are willing to try out Production Decomposition in their own interoperability test systems prior to its general release. You may install the alpha version using the InterSystems Package Manager: 
```
zpm "load https://github.com/intersystems/git-source-control -b production-change-control" 
```

Please keep in mind that this will install an unreleased branch of the extension that is undergoing active development! Any feedback that alpha testers have will be greatly appreciated.

[This document](https://github.com/intersystems/git-source-control/blob/production-change-control/docs/production-decomposition.md) contains more information about how to configure and use Production Decomposition.

**Comment by isc-bspead** [comment]
This is going to be a huge help, thank you @isc-pbarton!!
